.codeclimate.yml

```diff
@@ -1,18 +0,0 @@
-engines:
```

Comment:
Sorry. Let's restore this file until integration with codeclimate is not turned off.

Reply:
Test script is changed. `.codeclimate.yml` restored